notebooks/en/rag_with_sql_reranker.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<blockquote>This notebook will show you how to make a simple Retrieval Augmented Generation (RAG) system that draws on an SQL database instead of drawing information from a document store. </blockquote><p> </p><p>Suggestion for readability</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Changed to suggestion.